modules/build/src/main/scala/scala/build/preprocessing/directives/UsingResourcesDirectiveHandler.scala

```diff
@@ -26,19 +27,28 @@ case object UsingResourcesDirectiveHandler extends UsingDirectiveHandler {
   ): Either[BuildException, ProcessedUsingDirective] = either {
     val (virtualRootOpt, rootOpt) = Directive.osRootResource(cwd)
     val paths                     = DirectiveUtil.stringValues(directive.values, path, cwd)
-    val paths0                    = rootOpt.map(root => paths.map(_._1).map(os.Path(_, root)))
+    val paths0: Seq[os.Path] = value {
+      rootOpt.map(root => paths.map(_._1).map(os.Path(_, root))).toList.flatten
+        .map(validatePath)
+        .sequence.left.map(_.head)
```

Comment:
This discard errors (only keeps the first one). This shouldn't discard them:
```suggestion
        .sequence
        .left.map(CompositeBuildException(_))
```